src/lib.rs

```diff
@@ -86,32 +88,41 @@ fn unsafe_alloc_buffer(len: usize) -> Vec<u8> {
     ret
 }
 
+/// Enum representing an error code for `codeCopy`, `callDataCopy`, `externalCodeCopy`, and
```

Comment:
I would change this to: Enum representing an error code. Currently used by ...

Reply:
Fixed.